go-controller/pkg/network-controller-manager/network_controller_manager.go

```diff
@@ -300,24 +300,28 @@ func (cm *networkControllerManager) initDefaultNetworkController() error {
 func (cm *networkControllerManager) Start(ctx context.Context) error {
 	klog.Info("Starting the network controller manager")
 
-	// Make sure that the NCM zone matches with the Noruthbound db zone.
+	// Make sure that the NCM zone matches with the Northbound db zone.
 	// Wait for 300s before giving up
+	maxTimeout := 300 * time.Second
+	klog.Infof("Waiting up to %s for NBDB zone to match: %s", maxTimeout, config.Default.Zone)
```

Comment:
how does this log work for multi-zones? config.Default.Zone won't be used in those cases right?

Reply:
config.Default.Zone is not a default zone constant, but the value that is provided to ovnkube as config

Reply:
@trozet pointed out to me that this `config.Default.Zone` is not the default "global". It get's over-written. marking comment as resolved.